Library/Formula/osquery.rb

```diff
@@ -16,12 +16,11 @@ class Osquery < Formula
   depends_on :macos => :mavericks
 
   depends_on "cmake" => :build
-
-  depends_on "boost"
-  depends_on "gflags"
-  depends_on "openssl"
-  depends_on "rocksdb"
-  depends_on "thrift"
+  depends_on "boost" => :build
```

Comment:
We figured that might happen. That's very much because of how your build hosts library include path is configured. Added openssl back in as a distribution dependency.